### PR TITLE
Autocomplete updates (react-query refactor)

### PR DESCRIPTION
**Pull request by pfrazee**
- Updates the composer's web autocomplete to use the new hook
- Unifies the autocomplete hooks
- Drops the `fuse.js` usage. I did a side-by-side with the results and the fuzzy matching just added mostly-wrong guesses. We should let the backend do that behavior.
- Updates the mobile composer autocomplete to persist results while new ones are loading

### CODE REVIEW

File: src/view/com/composer/text-input/mobile/Autocomplete.tsx [hunk 1]
```diff
@@ -19,7 +21,13 @@ export const Autocomplete = observer(function AutocompleteImpl({
   const positionInterp = useAnimatedValue(0)
   const {getGraphemeString} = useGrapheme()
   const isActive = !!prefix
-  const {data: suggestions} = useActorAutocompleteQuery(prefix)
+  const {data: suggestions, isFetching} = useActorAutocompleteQuery(prefix)
+  const suggestionsRef = useRef<
```
Comment (gaearon) [inline src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: Why is this needed?

Comment (gaearon) [reply src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: To rephase, why is the code below not using `suggestions` directly?

Comment (estrattonbailey) [reply src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: Looks like it's so it renders the old results while new results are being fetched. I'll add a comment

Comment (estrattonbailey) [reply src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: Looks like it's working as intended without though 🤔

**Review by estrattonbailey — APPROVED** [review]
LGTM, one question on the ref

File: src/view/com/composer/text-input/mobile/Autocomplete.tsx (hunk 1 above)
Comment (pfrazee) [reply src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: Yep, to show old results while new ones are being fetched

Comment (gaearon) [reply src/view/com/composer/text-input/mobile/Autocomplete.tsx:25]: I see. The canonical pattern for this looks a bit different (writing to refs during render isn't supported by React) but by the time React adds a warning/error for this we'll probably rewrite this five times anyway so lg
